Network/Entries/Album+Mapping.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  Album+Mapping.swift
+//  CleanArchitectureRxSwift
+//
+//  Created by Andrey Yastrebov on 10.03.17.
+//  Copyright © 2017 sergdort. All rights reserved.
+//
+
+import Domain
+import ObjectMapper
+
+extension Album: ImmutableMappable {
+    
+    // JSON -> Object
+    public init(map: Map) throws {
+        title = try map.value("title")
+        uid = try map.value("id")
```

Comment:
Like here for example is it possible to say that I want `String` representation of `Int` ?